client/src/components/Suggestions.jsx

```diff
@@ -48,25 +50,33 @@ class Suggestions extends React.Component {
     this.scroller.current.scrollLeft -= 924;
   }
 
-  hoverIn(id) {
-    console.log(id);
+  hoverIn(id, e, index) {
```

Comment:
Where is the event being used here? Also, if you use it, you should probably name it `event` so it's clearer.